From: Ingo Saitz <ingo@debian.org>
Date: Sat, 29 Oct 2011 14:45:10 +0200
Subject: Drop LIBOBJS from configure.in

In order to rebuild the autotools files the configure.in must be changed.
According to the automake docs LIBOBJS is obsolete and can be deleted.

Closes: #263527
---
 configure.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index f38461c..3f8b6b6 100644
--- a/configure.in
+++ b/configure.in
@@ -377,7 +377,7 @@ fi
 AC_CHECK_FUNCS(setenv snprintf srandom)
 AC_REPLACE_FUNCS(strcasecmp strdup strstr)
 # Change extension, as we use libtool
-LIBOBJS="`echo $LIBOBJS|sed -e 's/\.o/\.lo/g'`"
+#LIBOBJS="`echo $LIBOBJS|sed -e 's/\.o/\.lo/g'`"
 
 AC_HEADER_EGREP(srandom,math.h,AC_DEFINE(SRANDOM_IN_MATH_H))
 
